bot/tradutor.py

```diff
@@ -0,0 +1,109 @@
+import requests
+
+import re
+
+class Tradutor:
+  filmes = {'i':"Ameaça Fantasma",
+            '1':"Ameaça Fantasma",
+            'ii':"Ataque dos Clones",
+            '2':"Ataque dos Clones",
+            'iii':"Vingança dos Sith",
+            '3':"Vingança dos Sith",
+            'iv':"A Nova Esperança",
+            '4':"A Nova Esperança",
+            '5':"O Império Contra-Ataca",
+            'v':"O Império Contra-Ataca",
+            '6':"Retorno do Jedi",
+            'vi':"Retorno do Jedi",
+            'vii':"O Despertar da Força",
+            '7':"O Despertar da Força",
+            'viii':"Os Últimos Jedi",
+            '8':"Os Últimos Jedi",
+            'ix':"A Ascensão Skywalker",
+            '9':"A Ascensão Skywalker",
+            'ro':"Rogue One: Uma História Star Wars",
+            'solo':"Solo: Uma História Star Wars"
+            }
+  def __init__(self):
+    #self.importa_local()
+    #self.importa_template()
+    self.raw = dict()
+    self.processed = dict()
+    self.importa_apendice()
+  
+  def importa_template(self):
+    predef = "https://starwars.fandom.com/pt/wiki/Predefini%C3%A7%C3%A3o:Tradu%C3%A7%C3%A3oSWW?action=raw"
+    r = requests.get(predef)
+    p = r.text.split("{{")
+    #Hoje dá para substituir isso por um p[3], mas não sei se daqui a alguns anos
+    #a arquitetura da predefinição seguirá assim.
+    p = [item for item in p if item.startswith('{1}}}}')][0]
+    p = p.split("\n|")[1:-1]
+    #Não vale a pena botar isso em um dicionário, você vai acabar tendo que iterar a lista toda de qualquer maneira.
+    self.dicio = dict()
+    for item in p:
+      k, v = item.split(" = ")
+      self.dicio[k] = v
+
+  def importa_apendice(self):
+    apendice = "https://starwars.fandom.com/pt/wiki/Star_Wars_Wiki:Ap%C3%AAndice_de_Tradu%C3%A7%C3%A3o?action=raw"
+    r = requests.get(apendice)
+    EhEntrada = lambda n: n.startswith("*")
```

Comment:
[PEP8 please](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)!

Reply:
Aliás, para quê serve essa função lambda? Ela não está sendo utilizada...

Reply:
Nem ela nem essa função ou a irmã mais nova o importa_local . O importa_apendice ficou aí só para caso vocês decidam mudar de ideia e voltar a exibição minimalista da tradução, o importa_local foi criado para que eu pudesse usar as funções na idle do python.

Se a comunidade eventualmente optar por dar rollback, o Tradutor.formata() perderá sua função dando lugar a um modesto .split(" = ")[1]